.github/workflows/release.yml

```diff
@@ -26,20 +26,28 @@ jobs:
     container: ${{ matrix.container }}
     strategy:
       matrix:
-        binary: [aarch64-apple-darwin, x86_64-manylinux2014, x86_64-manylinux2014-cuda117]
+        binary: [aarch64-apple-darwin, x86_64-manylinux2014, x86_64-manylinux2014-cuda117, x86_64-pc-windows-msvc]
         include:
           - os: macos-latest
             target: aarch64-apple-darwin
+            raw: tabby
             binary: aarch64-apple-darwin
           - os: ubuntu-latest
             target: x86_64-unknown-linux-gnu
+            raw: tabby
             binary: x86_64-manylinux2014
             container: quay.io/pypa/manylinux2014_x86_64
           - os: ubuntu-latest
             target: x86_64-unknown-linux-gnu
+            raw: tabby
             binary: x86_64-manylinux2014-cuda117
             container: sameli/manylinux2014_x86_64_cuda_11.7
             build_args: --features cuda
+          - os: windows-latest
+            target: x86_64-pc-windows-msvc
+            raw: tabby.exe
+            binary: x86_64-pc-windows-msvc.exe
```

Comment:
In the last commit, changed `x86_64-pc-windows-msvc` to `x86_64-pc-windows-msvc.exe`.
without this change, last step `Upload artifacts` could not find binary to upload but report no error.

Reply:
can you add a field named like `ext`, with linux / mac unset but on windows set to `.exe`, then you could construct string in later steps without doing if checks on target_os

Reply:
nice idea 👍